store/postgres/src/entities.rs

```diff
@@ -666,7 +456,7 @@ impl Connection<'_> {
         // 5 minutes
         const MIGRATION_CHECK_FREQ: u64 = 20;
 
-        if self.storage.needs_migrating() {
+        if false {
```

Comment:
`if false`?

Reply:
I didn't quite want to do away with the code for subgraph migrations when I wrote this; but commit 6e5424f3 in `lutter/shard` will do away with this nonsense